### PR TITLE
fix: a11y issues attention

### PR DESCRIPTION
**Pull request by felicia-haggqvist**
Fixes a11y issues raised in [WARP-320 ](https://nmp-jira.atlassian.net/browse/WARP-320):

Add role and aria-label to w-attention.vue
Add aria-describedby, tabIndex to Attention component + add onFocus hook to tooltip, callout and popover example to make them visible on focus. User is now able to tab in order to make them visible, as well as press ESC to close them:
Skärmavbild 2023-10-12 kl  14 15 45
Skärmavbild 2023-10-12 kl  14 16 00
Skärmavbild 2023-10-12 kl  14 16 06

### CODE REVIEW

**Review by BalbinaK — COMMENTED** [review]
After a discussion me and @felicia-haggqvist had, we decided to double-check with the accessibility team whether the aria-label should be set on both the wrapping div and the arrow div ([see comment here](https://docs.google.com/document/d/125D5kP5ZuqdEHC8nxkHWcIkjkHgkcTPA9RYwhU5-BUo/edit#heading=h.7ednrv77nze0)).
Also, a question was raised whether or not we should default that aria-label in the Attention component to be something like "${attentionType} speech bubble pointing ${arrowDirection}". It would of course be possible to override that but at least we'd make sure that by default the Attention component is visible and meaningful to users relying on assistive technologies. Pinging @Skadefryd as I think he had opinions on defaulting aria values in our components 😄

File: dev/pages/Attention.vue [hunk 1]
```diff
@@ -20,24 +20,94 @@ const popoverShowing = ref(false)
       <div class="space-y-16">
         <div>
           <h2>Tooltip</h2>
-          <w-box neutral class="h4" ref="tooltipTarget" @mouseenter="tooltipShowing = true;" @mouseleave="tooltipShowing = false">Hover over me</w-box>
-          <w-attention tooltip bottom :target-el="tooltipTarget ? tooltipTarget.$el : null" v-model="tooltipShowing">
+          <w-box
+            neutral
+            class="h4"
```
[the remark this answers is not in the document]
Comment (felicia-haggqvist) [reply dev/pages/Attention.vue:25]: Yes I agree! I'll change it! :)

File: components/attention/w-attention.vue [hunk 2]
```diff
@@ -1,84 +1,123 @@
 <script setup>
-  import { watch, computed, ref, onMounted, nextTick } from 'vue'
-  import { attention as ccAttention } from '@warp-ds/css/component-classes'
-  import { computePosition, flip, offset, shift, arrow } from '@floating-ui/dom'
+import { watch, computed, ref, onMounted, nextTick } from "vue";
```
Comment (magnuh) [inline components/attention/w-attention.vue:2]: We really need to have some tool for code style/standards in place. 😅  
We use single quotes... and I'm not really sure about the indentation? 🤔  But we should not really have to think about these kind of things. 😅  I guess there is a task lying around somewhere to fix this...? 🤷 😄

Comment (felicia-haggqvist) [reply components/attention/w-attention.vue:2]: But I can also adjust my prettier-settings, just need to know which code style/standards we have. I can change easily to single qoutes, but what kind of kindentation do we have? 😄

Comment (magnuh) [reply components/attention/w-attention.vue]: In Vue I think we only have indentation for template, and not for script and style (if any). So I guess your change about the indentation was correct. 😄 It is a bit confusing in Vue due to files having being created at different times with different code standards. 😅 I think I would like to do a quick run through of the whole repo and align things... 🙂 So you could just leave it as it is now I guess. 👍

**Review by BalbinaK — APPROVED** [review]
LGTM

**Pull request merged**